apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -206,29 +207,39 @@ fn send_key(device: &mut Device, key: keyboard::Key) -> Result<(), CommandError>
 fn handle_command(device: &mut Device, data: &[u8]) -> Result<(), CommandError> {
     let ButtonStatusCommand { button, action } = data.try_into()?;
 
-    let key: keyboard::Key;
+    let key: Keyboard;
     match action {
         Action::Pressed => match button {
             Button::Select => {
-                key = keyboard::Key::Enter;
+                key = uinput::event::Keyboard::Key(keyboard::Key::Enter);
             }
             Button::Left => {
-                key = keyboard::Key::Left;
+                key = uinput::event::Keyboard::Key(keyboard::Key::Left);
             }
             Button::Right => {
-                key = keyboard::Key::Right;
+                key = uinput::event::Keyboard::Key(keyboard::Key::Right);
             }
             Button::Up => {
-                key = keyboard::Key::Up;
+                key = uinput::event::Keyboard::Key(keyboard::Key::Up);
             }
             Button::Down => {
-                key = keyboard::Key::Down;
+                key = uinput::event::Keyboard::Key(keyboard::Key::Down);
+            }
+            Button::Help => key = uinput::event::Keyboard::Misc(keyboard::Misc::Help),
```

Comment:
Wondering if we can map these misc keys (`Help` and `Pause`) to regular keyboard keys so dev testing with a keyboard easier -- any downsides to doing that?

Reply:
Good point, I'll update to normal keyboard keys. I wanted to make these keys as specific as possible, since I don't love the arbitrary mapping of `[` to rate decrease, for example. Depending on the on-screen keyboard's implementation, I was also worried actual character keys might interfere with write-ins. But since we are using character keys for other functions anyway that's a moot point.

Reply:
Yeah, I've been a little worried about collisions -- might be worth figuring out a way to centralise all our keyboard shortcuts across apps. Fortunately don't have to worry about the `VirtualKeyboard` though, since it doesn't fire real key events.